mmvstr.c

```diff
@@ -669,15 +645,13 @@ vstring xlate(vstring sin,vstring table)
 
 
 /* Returns the ascii value of a character */
-long ascii_(vstring c)
-{
+long ascii_(const char *c) {
   return ((long)c[0]);
```

Comment:
The implementation hazardous, because for a non-ASCII character the result depends on the implementation defined signedness of char (signed or unsigned char, you cannot tell).  This is not something that has to do with your PR, but it caught my eye.